.magento.app.yaml

```diff
@@ -1,7 +1,9 @@
+# Copyright © Magento, Inc. All rights reserved.
```

Comment:
Not magento file - I guess we don't need copyright here.

Reply:
Fixed